<!doctype html>
<html>

  <head>
    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Vincent Fortuin</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <link rel="shortcut icon" type="image/png" href="./images/favicon.png"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">

  </head>
  <body>
    <div class="wrapper">

      <section>

<p><img style="padding: 0 15px; float: left;" src="./images/portrait_VF_2020_sq.jpg" height=190 width=190/> 
  <font size=6>Vincent Fortuin</font> <br>
        Ph.D. Candidate, Computer Science <br>
        <a href="https://ml.inf.ethz.ch/">ETH Zürich, Institute for Machine Learning</a> <br>
        <a href="https://bmi.inf.ethz.ch/">Biomedical Informatics group</a> <br>

        Email: fortuin at inf.ethz.ch <br></p>
        <p><a href="https://scholar.google.com/citations?user=XBlrYTIAAAAJ" target="_blank" style="padding-left: 0px"><img src="./images/scholar.png" height="24px" width="24px"></a>
        <a href="https://github.com/fortuin" target="_blank" style="padding-left: 20px"><img src="./images/github.svg" height="24px" width="24px"></a>
        <a href="https://twitter.com/vincefort" target="_blank" style="padding-left: 20px"><img src="./images/twitter.svg" height="24px" width="24px"></a>
        <a href="https://www.linkedin.com/in/vincent-fortuin-42426b134/" target="_blank" style="padding-left: 20px"><img src="./images/linkedin.svg" height="24px" width="24px"></a>
        <a href="https://medium.com/@vincefort" target="_blank" style="padding-left: 20px"><img src="./images/medium.svg" height="24px" width="24px"></a>
        </p>

<br>
<br>

<!-- <h1> Bio </h1> -->
<p align='justify'>I am a PhD student in Computer Science at <a href="https://ethz.ch/en.html">ETH Zürich</a> under the supervision of <a href="https://bmi.inf.ethz.ch/people/person/gunnar-raetsch/">Gunnar Rätsch</a> and <a href="https://las.inf.ethz.ch/krausea">Andreas Krause</a> and member of the <a href="https://bmi.inf.ethz.ch/">Biomedical Informatics group</a> as well as the <a href="https://math.ethz.ch/sfs/eth-foundations-of-data-science.html">ETH Center for the Foundations of Data Science</a>. I am supported by a <a href="https://datascience.ch/PhD-Fellows/">PhD fellowship</a> from the <a href="https://datascience.ch/">Swiss Data Science Center</a> and am also an <a href="https://ellis.eu/en/projects/priors-and-inference-for-deep-probabilistic-models">ELLIS PhD student</a>. My research focuses on the interface between deep learning and probabilistic modeling. I am particularly keen to develop models that are more interpretable and data efficient, following the Bayesian paradigm. To this end, I am mostly trying to improve deep probabilistic models with better priors and more efficient inference techniques.
</p>

<p align='justify'>
I did my undergraduate studies in Molecular Life Sciences at the <a href="https://www.uni-hamburg.de/en/">University of Hamburg</a>, where I worked on phylogeny inference for quickly mutating virus strains with <a href="https://www.zbh.uni-hamburg.de/en/personen/bm/atorda.html">Andrew Torda</a>. I then went to ETH Zürich to study Computational Biology and Bioinformatics (in a joint program with the <a href="https://www.uzh.ch/cmsssl/en.html">University of Zürich</a>), with a focus on systems biology and machine learning. My master's studies were suported by an <a href="https://ethz.ch/students/en/studies/financial/scholarships/excellencescholarship.html">ETH Excellence Scholarship</a>. My master's thesis was about the application of deep learning to gene regulatory network inference under supervision of <a href="https://imsb.ethz.ch/research/claassen.html">Manfred Claassen</a>, for which I received the <a href="https://ethz.ch/en/the-eth-zurich/education/awards/willi-studer-prize.html">Willi Studer Prize</a>. During my master's studies, I also spent some time in <a href="https://hannalabweb.weizmann.ac.il/">Jacob Hanna's group</a> at the <a href="https://www.weizmann.ac.il/pages/">Weizmann Institute of Science</a>, working on multiomics data analysis in stem cell research.
Within my current PhD studies, I visited and worked with <a href="http://www.stephanmandt.com/">Stephan Mandt</a> at the <a href="https://uci.edu/">University of California in Irvine</a> and <a href="http://cbl.eng.cam.ac.uk/Public/Turner/Turner">Richard Turner</a> at the <a href="https://www.cam.ac.uk/">University of Cambridge</a>.
Moreover, I completed internships at <a href="https://www.disneyresearch.com/">Disney Research Zürich</a>, working with <a href="https://studios.disneyresearch.com/people/romann-weber/">Romann Weber</a> on deep learning for natural language understanding in the <a href="https://studios.disneyresearch.com/artificial-intelligence/">Machine Intelligence and Data Science team</a>, and at <a href="https://www.microsoft.com/en-us/research/lab/microsoft-research-cambridge/">Microsoft Research Cambridge</a>, working with <a href="https://www.microsoft.com/en-us/research/people/kahofman/">Katja Hofmann</a> on uncertainty quantification in deep learning in the <a href="https://www.microsoft.com/en-us/research/group/game-intelligence/">Game Intelligence team</a>.
My <a href="https://en.wikipedia.org/wiki/Erd%C5%91s%E2%80%93Bacon_number">Erdös–Bacon number</a> is <a href="./sigma_graph_erdos_bacon/erdos_bacon.html">six</a>.
 </p>

<br>


<h1> Publications</h1> 

<ol>

<li> <b>On Disentanglement in Gaussian Process Variational Autoencoders</b>
<br>
Simon Bing, <b>Vincent Fortuin</b>, Gunnar Rätsch 
<br>
arXiv, 2021
<br>
[<a href="https://arxiv.org/abs/2102.05507">paper</a>][<a href="https://github.com/ratschlab/dgp-vae">code</a>]
</li>

<br>
<li> <b>Bayesian Neural Network Priors Revisited</b>
<br>
<b>Vincent Fortuin</b>, Adrià Garriga-Alonso, Florian Wenzel, Gunnar Rätsch, Richard E Turner, Mark van der Wilk, Laurence Aitchison 
<br>
AABI, 2021
<br>
[<a href="https://arxiv.org/abs/2102.06571">paper</a>][<a href="https://github.com/ratschlab/bnn_priors">code</a>]
</li>

<br>
<li> <b>Exact Langevin Dynamics with Stochastic Gradients</b>
<br>
Adrià Garriga-Alonso, <b>Vincent Fortuin</b>
<br>
AABI, 2021
<br>
[<a href="https://arxiv.org/abs/2102.01691">paper</a>]
</li>

<br>
<li> <b>Annealed Stein Variational Gradient Descent</b>
<br>
Francesco D'Angelo, <b>Vincent Fortuin</b>
<br>
AABI, 2021
<br>
[<a href="https://arxiv.org/abs/2101.09815">paper</a>]
</li>

<br>
<li> <b>Factorized Gaussian Process Variational Autoencoders</b>
<br>
Metod Jazbec, Michael Pearce, <b>Vincent Fortuin</b>
<br>
AABI, 2021
<br>
[<a href="https://arxiv.org/abs/2011.07255">paper</a>][<a href="https://github.com/metodj/FGP-VAE">code</a>]
</li>

<br>
<li> <b>Scalable Gaussian Process Variational Autoencoders</b>
<br>
Metod Jazbec, <b>Vincent Fortuin</b>, Michael Pearce, Stephan Mandt, Gunnar Rätsch
<br>
AISTATS, 2021
<br>
[<a href="https://arxiv.org/abs/2010.13472">paper</a>][<a href="https://github.com/ratschlab/SVGP-VAE">code</a>]
</li>

<br>
<li> <b>Sparse Gaussian Process Variational Autoencoders</b>
<br>
Matthew Ashman, Jonathan So, Will Tebbutt, <b>Vincent Fortuin</b>, Michael Pearce, Richard E Turner
<br>
arXiv, 2020
<br>
[<a href="https://arxiv.org/abs/2010.10177">paper</a>][<a href="https://github.com/MattAshman/sgpvae">code</a>]
</li>

<br>
<li> <b>PACOH: Bayes-Optimal Meta-Learning with PAC-Guarantees</b>
<br>
Jonas Rothfuss, <b>Vincent Fortuin</b>, Andreas Krause
<br>
ICML workshop on Lifelong Learning, 2020
<br>
[<a href="https://arxiv.org/abs/2002.05551">paper</a>][<a href="https://github.com/jonasrothfuss/meta_learning_pacoh">code</a>]
</li>

<br>
<li> <b>GP-VAE: Deep Probabilistic Time Series Imputation</b>
<br>
<b>Vincent Fortuin</b>, Dmitry Baranchuk, Gunnar Rätsch, Stephan Mandt
<br>
AISTATS, 2020
<br>
[<a href="https://arxiv.org/abs/1907.04155">paper</a>][<a href="https://github.com/ratschlab/GP-VAE">code</a>]
</li>

<br>
<li> <b>Conservative Uncertainty Estimation By Fitting Prior Networks</b>
<br>
Kamil Ciosek, <b>Vincent Fortuin</b>, Ryota Tomioka, Katja Hofmann, Richard E Turner
<br>
ICLR, 2020
<br>
[<a href="https://openreview.net/forum?id=BJlahxHYDS">paper</a>][<a href="https://github.com/microsoft/conservative-uncertainty-estimation-random-priors">code</a>]
</li>

<br>
<li> <b>Mixture-of-Experts Variational Autoencoder for clustering and generating from similarity-based representations</b>
<br>
Andreas Kopf, <b>Vincent Fortuin</b>, Vignesh Ram Somnath, Manfred Claassen
<br>
arXiv, 2019
<br>
[<a href="https://arxiv.org/abs/1910.07763">paper</a>]
</li>

<br>
<li> <b>DPSOM: Deep Probabilistic Clustering with Self-Organizing Maps</b>
<br>
Laura Manduchi, Matthias Hüser, Gunnar Rätsch, <b>Vincent Fortuin</b>
<br>
NeurIPS workshop on Machine Learning for Health, 2019
<br>
[<a href="https://arxiv.org/abs/1910.01590">paper</a>][<a href="https://github.com/ratschlab/dpsom">code</a>]
</li>

<br>
<li> <b>META^2: Memory-efficient taxonomic classification and abundance estimation for metagenomics with deep learning</b>
<br>
Andreas Georgiou, <b>Vincent Fortuin</b>, Harun Mustafa, Gunnar Rätsch
<br>
MLCB, 2019
<br>
[<a href="https://arxiv.org/abs/1909.13146">paper</a>][<a href="https://github.com/ag14774/META2">code</a>]
</li>

<br>
<li> <b>MGP-AttTCN: An Interpretable Machine Learning Model for the Prediction of Sepsis</b>
<br>
Margherita Rosnati, <b>Vincent Fortuin</b>
<br>
arXiv, 2019
<br>
[<a href="https://arxiv.org/abs/1909.12637">paper</a>][<a href="https://github.com/mmr12/MGP-AttTCN">code</a>]
</li>

<br>
<li> <b>Meta-Learning Mean Functions for Gaussian Processes</b>
<br>
<b>Vincent Fortuin</b>, Heiko Strathmann, Gunnar Rätsch
<br>
NeurIPS workshop in Bayesian Deep Learning, 2019
<br>
[<a href="https://arxiv.org/abs/1901.08098">paper</a>]
</li>

<br>
<li> <b>SOM-VAE: Interpretable Discrete Representation Learning on Time Series</b>
<br>
<b>Vincent Fortuin</b>, Matthias Hüser, Francesco Locatello, Heiko Strathmann, Gunnar Rätsch
<br>
ICLR, 2019
<br>
[<a href="https://arxiv.org/abs/1806.02199">paper</a>][<a href="https://github.com/ratschlab/SOM-VAE">code</a>]
</li>

<br>
<li> <b>Scalable Gaussian Processes on Discrete Domains</b>
<br>
<b>Vincent Fortuin</b>, Gideon Dresdner, Heiko Strathmann, Gunnar Rätsch
<br>
NeurIPS workshop on Bayesian Nonparametrics, 2018
<br>
[<a href="https://arxiv.org/abs/1810.10368">paper</a>]
</li>

<br>
<li> <b>On the Connection between Neural Processes and Gaussian Processes with Deep Kernels</b>
<br>
Tim GJ Rudner, <b>Vincent Fortuin</b>, Yee Whye Teh, Yarin Gal
<br>
NeurIPS workshop on Bayesian Deep Learning, 2018
<br>
[<a href="http://bayesiandeeplearning.org/2018/papers/128.pdf">paper</a>]
</li>

<br>
<li> <b>InspireMe: Learning Sequence Models for Stories</b>
<br>
<b>Vincent Fortuin</b>, Romann M Weber, Sasha Schriber, Diana Wotruba, Markus H Gross
<br>
AAAI, 2018
<br>
[<a href="https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/viewPaper/16100">paper</a>]
</li>

<br>
<li> <b>Supervised learning on synthetic data for reverse engineering gene regulatory networks from experimental time-series</b>
<br>
Stefan Ganscha, <b>Vincent Fortuin</b>, Max Horn, Eirini Arvaniti, Manfred Claassen
<br>
bioRxiv, 2018
<br>
[<a href="https://www.biorxiv.org/content/biorxiv/early/2018/06/27/356477.full.pdf">paper</a>]
</li>

</ol>

</p>

<h1> Invited Talks </h1>
<ul>
  <li> <i>March 2020</i>: <a href="https://www.bihealth.org/en/research/research-groups/roland-eils/">Robert Eils's lab</a> at the Charité (Berlin, Germany)
    <br>
  <li> <i>March 2020</i>: <a href="https://research.google/teams/brain/">Google Brain</a> (Berlin, Germany)
    <br>
  <li> <i>February 2020</i>: Machine Learning Clinic of the <a href="https://www.swissdataalliance.ch/">Swiss Data Alliance</a> (Zürich, Switzerland)
    <br>
  <li> <i>October 2019</i>: <a href="https://talks.cam.ac.uk/talk/index/133840">Machine Learning Seminar</a> at the University of Cambridge (Cambridge, UK)
    <br>
  <li> <i>September 2019</i>: <a href="https://amazon.jobs/en/landing_pages/Cambridge">Amazon Research</a> (Cambridge, UK)
    <br>
  <li> <i>September 2019</i>: <a href="https://www.microsoft.com/en-us/research/lab/microsoft-research-cambridge/">Microsoft Research</a> (Cambridge, UK)
    <br>
  <li> <i>August 2019</i>: <a href="https://www.bayer.com/en/research-and-development.aspx">Bayer Research</a> (Berlin, Germany)
    <br>
  <li> <i>May 2019</i>: <a href="https://paige.ai/">PAIGE.AI</a> (New York City, USA)
    <br>
  <li> <i>February 2019</i>: <a href="https://sites.google.com/site/boumedienehamzi/symposium-on-machine-learning-for-dynamical-systems_2019">Symposium on Machine Learning and Dynamical Systems</a> (London, UK)
    <br>
  <li> <i>November 2018</i>: <a href="https://www.idiap.ch/workshop/smld2018">Swiss Machine Learning Day</a> (Lausanne, Switzerland)
   
</li>
</ul>

<h1> Teaching </h1>
<ul>
	<li> <i>January 2021</i>: Leading a tutorial on Deep Generative Models for Representation Learning at the <a href="https://www.ds3-datascience-polytechnique.fr/">Data Science Summer School</a> (Paris, France)
    <br>
  <li> <i>March 2020</i>: Leading a tutorial on Machine Learning for Healthcare at the Seminar on Digital Health by the <a href="https://www.studienstiftung.de/en/">German Academic Scholarship Foundation</a> (Berlin, Germany)
    <br>
  <li> <i>Spring 2018, 2019, 2020</i>: Teaching Assistant for <a href="https://las.inf.ethz.ch/teaching/introml-s18">Introduction to Machine Learning</a> at ETH Zürich
    <br>
  <li> <i>Fall 2018, 2019</i>: Teaching Assistant for <a href="https://las.inf.ethz.ch/teaching/pai-f18">Probabilistic Artificial Intelligence</a> at ETH Zürich
   
</li>
</ul>

<h1> Service </h1>
<ul>
<li> <i>Conference Reviewer</i>: NeurIPS, ICML, ICLR, AISTATS, AAAI, ECML-PKDD
<li> <i>Journal Reviewer</i>: Machine Learning, Bayesian Analysis
<li> <i>Workshop Organizer</i>: <a href="https://ellis.eu/covid-19">ELLIS against COVID 2020</a>
</ul>




      </section>

        <div>
  <p> <i>Template Design by <a href="http://aditya-grover.github.io"> Aditya Grover</a>. </i></p>
</div>


    </div>


        
  </body>



</html>
